src/main/java/com/commercetools/sync/producttypes/helpers/AttributeDefinitionReferenceResolver.java

```diff
@@ -112,12 +112,14 @@ public AttributeDefinitionReferenceResolver(@Nonnull final ProductTypeSyncOption
         final String resourceKey;
         try {
             resourceKey = getKeyFromResourceIdentifier(typeReference);
-        } catch (ReferenceResolutionException exception) {
+
+        } catch (ReferenceResolutionException referenceResolutionException) {
```

Comment:
What is the point of your change in here, it does not seem necessary to me. You are not improving anything in here, the handler was also simple and easy to read before.

Reply:
Would base on what implemented in master branch, so I would rollback this change.